app/pulp/app/tasks/orphan.py

```diff
@@ -0,0 +1,14 @@
+from celery import shared_task
+
+from pulp.app import models
+from pulp.tasking.tasks import UserFacingTask
+
+
+@shared_task(base=UserFacingTask)
+def delete_all():
+    """
+    Delete all orphan content.
+    This task removes content from the filesystem as well.
+    """
+    orphans = models.Content.objects.exclude(id__in=models.RepositoryContent.objects.all())
```

Comment:
Does `orphans = Content.objects.filter(repositories=None)` work as you'd expect?